docs/usage/key-concepts/automerge.md

```diff
@@ -73,7 +73,7 @@ Non-major updates in SemVer ecosystems shouldn't have breaking changes (if they
   "packageRules": [
     {
       "matchUpdateTypes": ["minor", "patch"],
-      "matchCurrentVersion": "!/^0/",
```

Comment:
I'm not sure if this will work for all managers though, might need to revert this?

Reply:
Actually it does work for all, because it's using regex. I prefer to keep it

Reply:
I've reverted this file to the original file.